src/Persistence.Shared/Folders.cs

```diff
@@ -48,7 +48,7 @@ public static string GetPackageFilePath(string filename)
 			return Foundation.NSBundle.MainBundle
 				.GetUrlForResource(Path.GetFileNameWithoutExtension(uri.LocalPath), Path.GetExtension(uri.LocalPath))
 				.AbsoluteString;
-#elif WINDOWS_UWP
```

Comment:
since it's shared, it should have both WINDOWS_UWP and WINUI

Reply:
updated condition